navigation/AppNavigator.js

```diff
@@ -20,7 +21,7 @@ const AuthNavigator = () => {
   });
 
   return (
-    <Stack.Navigator initialRouteName="Account">
+    <Stack.Navigator initialRouteName="Question">
```

Comment:
Should our initial route be Question? I know we're in progress on the nav and hooking everything up together but I think this may be problematic for @JaycobDuffel